.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     "array-callback-return": "off",
     "linebreak-style": "off",
     "no-console": 1,
+    'max-len': "off",
```

Comment:
@alibombaye @kat1906 is this ok to turn off? Seems like a max length of 100 is unnecessary...